src/main/java/org/worshipsongs/importer/SongParser.java

```diff
@@ -64,4 +88,75 @@ public String findMatchingData(String input, String attributeName)
         }
         return matchingData;
     }
+
+    public String getXmlLyrics(String lyrics, String verseOrder)
+    {
+        this.lyrics = lyrics;
+        this.verseOrder = verseOrder;
+
+        Writer out = new StringWriter();
+        try {
+            DocumentBuilderFactory docFactory = DocumentBuilderFactory.newInstance();
+            DocumentBuilder docBuilder = docFactory.newDocumentBuilder();
+            Document document = docBuilder.newDocument();
+
+            getSongTag(document);
+
+            Transformer transformer = TransformerFactory.newInstance().newTransformer();
+            document.setXmlStandalone(true);
+            transformer.transform(new DOMSource(document), new StreamResult(out));
+
+            System.out.print("Data:"+out.toString());
+        } catch (ParserConfigurationException pce) {
+            pce.printStackTrace();
+        } catch (TransformerException tfe) {
+            tfe.printStackTrace();
+        }
+        return out.toString();
+    }
+
+    public void getSongTag(Document document)
+    {
+        Element song = document.createElement("song");
+        Attr version = document.createAttribute("version");
+        version.setValue("1.0");
+        song.setAttributeNode(version);
+        document.appendChild(song);
+
+        getLyricsTag(song, document);
+    }
+
+    public void getLyricsTag(Element song, Document document)
```

Comment:
Do we need to define "song" element as a method parameter? Consider returning "lyrics" element instead.